src/player.ts

```diff
@@ -159,6 +159,11 @@ export default class Player extends EventEmitter {
     return building === Building.GaiaFormer ? (this.data.gaiaformers - this.data.gaiaformersInGaia) : this.board.buildings[building].income.length;
   }
 
+  get ownedPlanets(): GaiaHex[] {
+    return this.data.occupied.filter(hex => hex.data.planet !== Planet.Empty && hex.colonizedBy(this.player)
```

Comment:
Maybe `this.data.occupied.filter(hex => hex.data.planet !== Planet.Empty && hex.isMainOccupier(this.player))`

And add `isMainOccupier` to gaia hex. It can just return `this.colonizedBy(player) && this.additionalMine !== player`. No need to test against specific factions